packages/slate/src/models/range.js

```diff
@@ -240,6 +242,46 @@ class Range extends Record(DEFAULTS) {
     return !this.isSet
   }
 
+  /**
+   * Get the anchor key.
+   *
+   * @return {String}
+   */
+
+  get anchorKey() {
```

Comment:
We need to put deprecation warnings here, and encourage people to use `this.anchor.key` instead I think.